demo/web/all_demos.dart

```diff
@@ -0,0 +1,404 @@
+/// [GoogleSigninDemo]
+/// [GoogleStreetviewPanoDemo]
+/// [GoogleUrlShortenerDemo]
+/// [GoogleYoutubeDemo]
+/// [GoogleYoutubeUploadDemo]
+/// [IronA11yAnnouncerDemo]
+/// [IronA11yKeysDemo]
+/// [IronA11yKeysBehaviorDemo]
+/// [IronAjaxDemo]
+/// [IronAutogrowTextareaDemo]
+/// [IronBehaviorsDemo]
+/// [IronCheckedElementBehaviorDemo]
+/// [IronCollapseDemo]
+/// [IronComponentPageDemo]
+/// [IronDocViewerDemo]
+/// [IronDropdownDemo]
+/// [IronFitBehaviorDemo]
+/// [IronFlexLayoutDemo]
+/// [IronFormDemo]
+/// [IronFormElementBehaviorDemo]
+/// [IronIconDemo]
+/// [IronIconsDemo]
+/// [IronIconsetDemo]
+/// [IronIconsetSvgDemo]
+/// [IronImageDemo]
+/// [IronInputDemo]
+/// [IronJsonpLibraryDemo]
+/// [IronLabelDemo]
+/// [IronListDemo]
+/// [IronListCollapseDemo]
+/// [IronListExternalContentDemo]
+/// [IronListSelectionDemo]
+/// [IronLocalstorageDemo]
+/// [IronMediaQueryDemo]
+/// [IronMenuBehaviorDemo]
+/// [IronMetaDemo]
+/// [IronOverlayBehaviorDemo]
+/// [IronPagesDemo]
+/// [IronRangeBehaviorDemo]
+/// [IronResizableBehaviorDemo]
+/// [IronSelectorDemo]
+/// [IronSignalsDemo]
+/// [IronValidatableBehaviorDemo]
+/// [IronValidatorBehaviorDemo]
+/// [MarkedElementDemo]
+/// [NeonAnimationCardDemo]
+/// [NeonAnimationDeclarativeDemo]
+/// [NeonAnimationDocDemo]
+/// [NeonAnimationDropdownDemo]
+/// [NeonAnimationGridDemo]
+/// [NeonAnimationListDemo]
+/// [NeonAnimationLoadDemo]
+/// [NeonAnimationReprojectionDemo]
+/// [NeonAnimationTilesDemo]
+/// [PaperBadgeDemo]
+/// [PaperBehaviorsDemo]
+/// [PaperButtonDemo]
+/// [PaperCardDemo]
+/// [PaperCheckboxDemo]
+/// [PaperDialogDemo]
+/// [PaperDialogBehaviorDemo]
+/// [PaperDialogScrollableDemo]
+/// [PaperDrawerPanelDemo]
+/// [PaperDropdownMenuDemo]
+/// [PaperFabDemo]
+/// [PaperHeaderPanelDemo]
+/// [PaperIconButtonDemo]
+/// [PaperInputDemo]
+/// [PaperItemDemo]
+/// [PaperListboxDemo]
+/// [PaperMaterialDemo]
+/// [PaperMenuDemo]
+/// [PaperMenuButtonDemo]
+/// [PaperProgressDemo]
+/// [PaperRadioButtonDemo]
+/// [PaperRadioGroupDemo]
+/// [PaperRippleDemo]
+/// [PaperScrollHeaderPanelDemo]
+/// [PaperSliderDemo]
+/// [PaperSpinnerDemo]
+/// [PaperStylesDemo]
+/// [PaperTabsDemo]
+/// [PaperToastDemo]
+/// [PaperToggleButtonDemo]
+/// [PaperToolbarDemo]
+/// [PaperTooltipDemo]
+@PolymerRegister('all-demos')
+class AllDemos extends PolymerElement {
+  AllDemos.created() : super.created();
+  List<DemoElementItem> __demos;
+  @property List<DemoElementItem> get demos => __demos;
+  set demos(value) {}
+
+  @property
+  DemoElementItem selected;
+
+  void ready() {
+    __demos = demoElements.map((name) => new DemoElementItem(name)).toList()
+        as List<DemoElementItem>;
+    notifyPath('demos', __demos);
+    _demoList.selectItem(convertToJs(demos.first));
+    _loadDemo(demos.first);
+  }
+
+  IronList get _demoList => $['demolist'] as IronList;
+
+  @reflectable
+  void demoClickHandler(dom.Event event, [_]) {
+    final item = convertToDart(_demoList.jsElement.callMethod(
```

Comment:
Mind referencing https://github.com/dart-lang/polymer_elements/issues/90 in a comment above this?